src/main/java/com/microsoft/sqlserver/jdbc/DataTypes.java

```diff
@@ -230,7 +233,7 @@ enum GetterConversion {
 
         CHARACTER(SSType.Category.CHARACTER, EnumSet.of(JDBCType.Category.NUMERIC, JDBCType.Category.DATE,
```

Comment:
Do we need JDBCType.Category.VECTOR in GetterConversion for the CHARACTER type, and vice versa?